lib/splitclient-rb/clients/split_client.rb

```diff
@@ -164,6 +178,33 @@ def sanitize_split_names(calling_method, split_names)
 
     private
 
+    def fix_properties(properties)
```

Comment:
could you please change the name of the function to something like `validate_properties`? thanks!

Reply:
changed